web/src/locales/en/data.json

```diff
@@ -133,7 +133,8 @@
     "Vectors": "Vectors",
     "Videos": "Videos",
     "View": "View",
-    "Wordsets": "Wordsets"
+    "Wordsets": "Wordsets",
+    "Livechat": "Livechat"
```

Comment:
i18n is not added manually, just don't mess with i18n in this PR

Reply:
OK, I've deleted my changes in i18n json files.